File: C++/1159.cpp
#include<iostream>

using namespace std;

int main(){
    int entrada;
    int soma, total = 0;
    while(true){
        cin >> entrada;
        total = 0;
        soma = entrada;
        if(entrada == 0){
            break;
        }
        if(entrada % 2 != 0){
            soma = soma + 1;
        }
        for(int i = 0; i < 5; i++){
            total = total + soma;
            soma = soma + 2;
        }
        cout << total << endl;
    }

    return 0;
}

File: JavaScript/1159.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');
let entrada, i = 0;
let total, soma;

while(true){
    entrada = parseInt(lines[i]);
    soma = entrada;
    total = 0;
    if(entrada === 0){
        break;
    }
    if(soma % 2 !=0){
        soma = soma +1;
    }

    for(let j = 0; j < 5; j++){
        total = total + soma;
        soma = soma + 2;
    }

    console.log(total)
    i++;
}